src/main/java/com/okta/sdk/clients/AuthApiClient.java

```diff
@@ -141,9 +146,16 @@ public AuthResult resetPassword(String stateToken, String relayState, String new
     }
 
     public AuthResult forgotPassword(String username, String relayState) throws IOException {
+        return forgotPassword(username, null, relayState);
+    }
+
+    public AuthResult forgotPassword(String username, FactorType factorType, String relayState) throws IOException {
```

Comment:
I think we'll ultimately make the relayState optional, since it's not used by everyone. So considering the addition of future methods, I think it makes sense to keep it here.